.gitignore

```diff
@@ -1,2 +1,3 @@
 .DS_Store
-.idea
```

Comment:
From the gitignore, it looks like we always wanted to ignore the `.idea` folder.
This folder can only be correctly ignored after we remove it from git. Otherwise git will keep tracking the changes.